Run pre-commit's yml hook on the entire tree

superm1 opened the pull request:
In order for pre-commit to be useful, everything needs to meet a common baseline.

# Motivation
I want to be able to rely upon pre-commit to help fixup style when submitting a PR. I also think it is worth having a pre-commit hook for the project for CI to mention style issues later. I do this in another project (fwupd).

# Technical Details
I ran pre-commit run --all-files and this is the result for just the YML files.

# Test Plan
No testing is done as it's style only changes.

nunnikri reviewed (approved):
Approve for rocm-core changes

rakesroy commented:
No need of stress testing as it is not hip-tests testcode change.

superm1 committed:
Run pre-commit's yml hook on the entire tree

In order for pre-commit to be useful, everything needs to meet a common
baseline.

Signed-off-by: Mario Limonciello (AMD) <superm1@kernel.org>
